### PR TITLE
Fixed errant text and comma in docs

### PR DESCRIPTION
**Pull request by ysuarez**
Thanks to @bseeger for sending me link for original version of this text.

### CODE REVIEW

File: docs/install/install-local-new.md [hunk 1]
```diff
@@ -181,7 +181,8 @@ You now have the current ISLE project code checked into git as a foundation to m
 * Follow the in-line instructions to add your project's name to the appropriate areas.
     * Once finished, save and close the file.
 
-* _Using the same open terminal:_, navigate to "/pathto/yourprojectnamehere/scripts/proxy/ssl-certs/"
+* _Using the same open terminal:_
```
Comment (bseeger) [inline docs/install/install-local-new.md]: Hi - I think you can change this back and just take out the comma. The first line `navigate to...` is meant as a summary with the `cd ... ` as how to do it.

Comment (ysuarez) [reply docs/install/install-local-new.md]: @bseeger no problem, I will change it back. Should I keep the comma right after the semicolon?

Comment (dwk2) [reply docs/install/install-local-new.md]: good catch: please remove the comma after the semicolon.

Comment (dwk2) [reply docs/install/install-local-new.md]: resolving this conversation as we need to push this into the docs for islandoracon.

**Comment by bseeger** [comment]
@ysuarez  -  I went and made the change. I apologize for not leaving it for you directly, but we're under a bit of a time crunch preparing for the workshop on Monday.

**Pull request merged**